.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,17 @@
+name: 💡 Feature Request
+description: Suggest a feature request
+title: "feat:"
+labels: ["feature request"]
```

Comment:
I don't think we need this label, we will add a specific label depending on the feature

Reply:
Oops, will fix this asap